airflow/datasets/__init__.py

```diff
@@ -159,6 +161,23 @@ def iter_datasets(self) -> Iterator[tuple[str, Dataset]]:
         raise NotImplementedError
 
 
+@attr.define()
+class DatasetAlias:
+    """A represeation of datasets to be created during the runtime."""
+
+    name: str
+
+    def __str__(self) -> str:
+        return f"DatasetAlias(name={self.name})"
```

Comment:
Isn’t `__str__` generation a built-in feature from attrs?

Reply:
I added this to all the newly added classes for quick testing. Forget to clean up 🤦‍♂️ Will remove it

Reply:
Removed